test/integration/analysis-controller-existing-status/00-assert.yaml

```diff
@@ -0,0 +1,14 @@
+apiVersion: metrics.keptn.sh/v1alpha3
+kind: Analysis
+metadata:
+  name: analysis-sample
+  namespace: testy
```

Comment:
with https://github.com/keptn/lifecycle-toolkit/pull/2105 being merged, can we remove the hardcoded namespace?

Reply:
to remove the hardcoded namespace I could use a sed command to generate the first assertion, since the namespace is included in the [object key](ad2ee2c6cc15452a86eecc353afddf00894967376c5ecb6d709085f8f1d77dd5R11)... I was just lazy but can try it